Mathlib/FieldTheory/Finite/Basic.lean

```diff
@@ -166,6 +166,32 @@ theorem cast_card_eq_zero : (q : K) = 0 := by
   simp
 #align finite_field.cast_card_eq_zero FiniteField.cast_card_eq_zero
 
+theorem cast_subgroup_of_units_card_ne_zero [DecidableEq K]
+    (G : Subgroup (Units (K))) [Fintype G] : (Fintype.card G : K) ≠ 0 := by
```

Comment:
I'm confused by this statement; isn't the cardinality of _any_ subgroup non-zero because it contains 1?

Reply:
Or maybe you're interested in infinite fields as well? Let me see if anything fails when I remove the finiteness from the field.

Reply:
Oh, I just totally missed the cast!